<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgRecord</class>
 <widget class="QDialog" name="DlgRecord">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>798</width>
    <height>603</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="btn_acc_reject">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>560</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QTableWidget" name="tbl_records">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>420</y>
     <width>651</width>
     <height>141</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_refresh">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>20</y>
     <width>101</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Refresh</string>
   </property>
  </widget>
  <widget class="QWidget" name="videow" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>651</width>
     <height>371</height>
    </rect>
   </property>
  </widget>
  <widget class="QSlider" name="video_silder">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>390</y>
     <width>651</width>
     <height>29</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btn_acc_reject</sender>
   <signal>accepted()</signal>
   <receiver>DlgRecord</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_acc_reject</sender>
   <signal>rejected()</signal>
   <receiver>DlgRecord</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
